jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/StepsRunner.java

```diff
@@ -228,7 +297,7 @@ private void pushApplicationLayers() {
                         Verify.verifyNotNull(results.applicationLayers))));
   }
 
-  private void pushImage() {
+  private void pushImages() {
```

Comment:
I think this should remain `pushImage`? Do we really consider multiple tags to be multiple images?